pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -150,6 +151,9 @@ class ModelCheckpoint(Callback):
             if ``mode`` is none of ``"min"`` or ``"max"``.
         ValueError:
             If ``trainer.save_checkpoint`` is ``None``.
+        UserWarning:
```

Comment:
I think raises is used only for errors:
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format